app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -153,7 +157,10 @@ class TopicLessonsFragmentTest {
   @Inject lateinit var storyProgressTestHelper: StoryProgressTestHelper
   @Inject lateinit var fakeOppiaClock: FakeOppiaClock
   @Inject lateinit var fakeAccessibilityService: FakeAccessibilityService
-  @Inject lateinit var explorationCheckpointTestHelper: ExplorationCheckpointTestHelper
+  @Inject lateinit var spotlightStateController: SpotlightStateController
+
+  @Inject
+  lateinit var explorationCheckpointTestHelper: ExplorationCheckpointTestHelper
```

Comment:
```suggestion
  @Inject lateinit var explorationCheckpointTestHelper: ExplorationCheckpointTestHelper
```

For consistency. Though, alternatively, perhaps just remove this & readd the original injection line that was removed about the spotlightStateController injection?

Reply:
doing the suggested approach